packages/material-ui-codemod/src/v5.0.0/box-sx-prop.test/actual.js

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+
+export default function BoxComponent() {
+  return (
+    <Box border="1px dashed grey" p={[2, 3, 4]}>
+      <Box component="span" clone p={{ xs: 2, sm: 3, md: 4 }} m={2} border="1px dashed grey">
+        <Button component="span">Save</Button>
+      </Box>
```

Comment:
As we are going to merge https://github.com/mui-org/material-ui/pull/23454 soon and add deprecation in v4, should we add two more scenarios:
```diff
- <Box border="1px dashed grey" p={[2, 3, 4]} css={{ m: 1 }}>
+ <Box sx={{ border: "1px dashed grey", p: [2, 3, 4], m: 1 }}>

- <Box border="1px dashed grey" p={[2, 3, 4]} sx={{ m: 1 }}>
+ <Box sx={{ border: "1px dashed grey", p: [2, 3, 4], m: 1 }}>
```

Reply:
For the first scenario, I didn't want to add that until #23454 is merged. For the second, the `sx` prop doesn't exist in v4.

Reply:
Alright, let’s do that after we have the breaking change and deprecation PRs in.

Reply:
So, #23454 is merged. I'd like to address it as a follow-up PR though. This one was hard enough! 😅

Reply:
Sure, I will work tomorrow on deprecation for v4 and we can update it after that one is merged too :)